icons/group.svg

```diff
@@ -9,10 +9,10 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M3 7V5c0-1.1.9-2 2-2h2" />
-  <path d="M17 3h2c1.1 0 2 .9 2 2v2" />
-  <path d="M21 17v2c0 1.1-.9 2-2 2h-2" />
-  <path d="M7 21H5c-1.1 0-2-.9-2-2v-2" />
-  <rect width="7" height="5" x="7" y="7" rx="1" />
-  <rect width="7" height="5" x="10" y="12" rx="1" />
+  <path d="M17 3h2a2 2 0 0 1 2 2v2" />
+  <path d="M21 17v2a2 2 0 0 1-2 2h-2" />
+  <path d="M3 7V5a2 2 0 0 1 2-2h2" />
+  <path d="M7 21H5a2 2 0 0 1-2-2v-2" />
```

Comment:
@jguddas: you ought not have closed it, these optimisations would've been nice anyway. 😅

Reply:
We can run the optimization script on all the icons at once at some point in the future.